libs/core/src/global/styles/tokens/tokens.json

```diff
@@ -751,6 +819,154 @@
           }
         }
       }
+    },
+    "heading-1": {
+      "value": {
+        "fontWeight": "{font-weight.bold}",
+        "fontSize": "{font-size.250}",
+        "lineHeight": "{line-height.275}",
+        "letterSpacing": "{letter-spacing.md}"
+      },
+      "type": "typography"
```

Comment:
We should be able to add the font family in here as well in TS.

Reply:
@anechol all should be set to `{font-family.circular}` now. Great catch!